### PR TITLE
Split changes.md by major version

### PR DESCRIPTION
**Pull request by s2quake**
✂️ 📑 https://github.com/planetarium/libplanet/issues/3647

### CODE REVIEW

**Review by riemannulus — COMMENTED** [review]
`Changes` is not a project directory, we use the first character upper-case only for `Project` directory. So I suggest `changes` instead of `Changes`.

**Comment by riemannulus** [comment]
How about `changes/v1.md` instead of `changes/CHANGES_1.md`? IMO `v1` represents more semantic meaning.

File: CHANGES.md [hunk 1]
```diff
@@ -194,8834 +194,10 @@ Released on January 22, 2024.
 [#3626]: https://github.com/planetarium/libplanet/pull/3626
 
 
-Version 3.9.6
+Older changes
```
Comment (riemannulus) [inline CHANGES.md:197]:
```suggestion
Previous version changes
```

**Review by riemannulus — DISMISSED** [review]

**Comment by s2quake** [comment]
https://github.com/planetarium/libplanet/assets/887680/b36a0892-38b1-4b5b-ac24-dda8e859bb52

@riemannulus each `previous version page`s are displayed.

**Review by riemannulus — APPROVED** [review]
Cool

**Pull request merged**